### PR TITLE
chore(build): use inline sources in sourcemaps; don't package sources separately

### PR DESCRIPTION
**Pull request by cpcallen**
## The basics

- [X] I branched from develop
- [X] My pull request is against develop
- [X] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)
- [X] I ran `npm run format` and `npm run lint`

### Resolves

`tsc` compile failures for developers whose build pipeline is misconfigured in a way that causes it to ingest `.ts` source files from `node_modules/blockly/core/`—somehow without also ingesting `core/any_aliases.ts`—resulting in compile errors due to `AnyDuringMigration`.  (See e.g. item #2 of #6358.)

### Proposed Changes

Don't package the source `.ts` files; instead, include sources inline in the `.js.map` sourcemaps.

#### Behaviour Before Change

`.ts` files are copied from `core/`, `blocks/` and `generators/` to `dist/`.

Sourcemap (`.js.map`) files contain a two-byte `sourceRoot` property.

#### Behaviour After Change

`.ts` files are not packaged.

Sourcemap files contain a large `sourcesContent` property (~2MB for `blockly_compressed.js.map), containing all of the original source.

### Reason for Changes

@btw17 points out that it is [generally not advised to package `.ts` sources along with compiled code in NPM packages]https://stackoverflow.com/a/57534667/4969945), and empirically it does appear to be causing problem for some of our developers.

### Test Coverage

Manually verified that sourcemaps with inline sources work correctly in compiled and uncompiled mode.

### Additional Information

I sent email to @samelhusseini to get clarity about why sources were originally included in the Blockly NPM package.  I think it worth waiting for a response before merging this PR.

### CODE REVIEW

**Review by maribethb — APPROVED** [review]
LGTM assuming we have sam's blessing. were you able to test the resulting sourcemaps work as expected in a plugin?

**Comment by cpcallen** [comment]
> LGTM assuming we have sam's blessing.

I will assign to Rachel to submit when we have a reply (or she decides we don't need one).

> were you able to test the resulting sourcemaps work as expected in a plugin?

That is one configuration I did not test, but I did test loading in compiled mode which uses the same `.js.map` files so I'm 95% sure this will work.

**Pull request merged**
